node/full_client.go

```diff
@@ -486,7 +486,11 @@ func (c *FullClient) Commit(ctx context.Context, height *int64) (*ctypes.ResultC
 	if err != nil {
 		return nil, err
 	}
-	commit := com.ToABCICommit(heightValue, b.Hash())
+
+	// we only have one centralize sequencer = 1 validators
+	val := b.SignedHeader.Validators.Validators[0].Address
+	commit := com.ToABCICommit(heightValue, b.Hash(), val, b.SignedHeader.Time())
```

Comment:
Rollkit doesn't have validators. At the moment, we want to get the centralized sequencer from the genesis, and keep track of their pubkey in `Header.ProposerPubkey`.

Validators will be carefully added back for IBC compatibility in the near future

Reply:
not so sure yet. can we discuss after https://github.com/rollkit/rollkit/pull/1368 is merged?

Reply:
Ok nevermind, we're keeping the Validators field.
We might merge this!

Reply:
@S1nus yes, easier to keep rollup IBC compatible when we have valset info in rollkit header